<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGUI">
      <attribute name="title">
       <string>GUI</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutTabGUI" columnstretch="0,1,0,0">
       <item row="0" column="0">
        <widget class="QLabel" name="labelLanguage">
         <property name="text">
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxLanguage">
         <property name="toolTip">
          <string>Select the GUI language (requires a restart to take effect!)</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>German (de)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spanish (es)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Greek (el)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>French (fr)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Italian (it)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Polish (pl)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Portuguese (pt)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Romanian (ro)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Swedish (sv)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English (us)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxStyle">
         <property name="toolTip">
          <string>Select the GUI style</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelAppFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <spacer name="verticalSpacerGUI">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>331</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QFontComboBox" name="fontComboBoxAppFont">
         <property name="toolTip">
          <string>Application font</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labelAppFontSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBoxAppFontSize">
         <property name="toolTip">
          <string>Application font size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelLogFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Log font</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFontComboBox" name="fontComboBoxLogFont">
         <property name="toolTip">
          <string>Log font</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="labelLogFontSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QSpinBox" name="spinBoxLogFontSize">
         <property name="toolTip">
          <string>Log font size</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayoutGuiFlags">
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBoxMaximizeWindows">
           <property name="toolTip">
            <string>Maximize new project or script windows</string>
           </property>
           <property name="text">
            <string>Maximize windows</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxShowProjectHelp">
           <property name="toolTip">
            <string>Make sure project help texts are shown</string>
           </property>
           <property name="text">
            <string>Display project help texts</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxNativeFileDialogs">
           <property name="toolTip">
            <string>Use the platform's native file dialogs or the Qt built-in ones</string>
           </property>
           <property name="text">
            <string>Use native file dialogs</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBoxLogChannelNames">
           <property name="toolTip">
            <string>Include the names of the I/O channels (stdout / stderr) when logging CHDMAN output</string>
           </property>
           <property name="text">
            <string>Log channel names</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelEditorFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Editor font</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QFontComboBox" name="fontComboBoxEditorFont">
         <property name="toolTip">
          <string>Editor font</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="labelEditorFontSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QSpinBox" name="spinBoxEditorFontSize">
         <property name="toolTip">
          <string>Editor font size</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPaths">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutTabPaths">
       <item row="1" column="0">
        <widget class="QLabel" name="labelChdmanBinary">
         <property name="text">
          <string>CHDMAN binary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEditChdmanBinary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Path to the CHDMAN binary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="toolButtonBrowseChdmanBinary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="icon">
          <iconset resource="qchdman.qrc">
           <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <spacer name="verticalSpacerPaths">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="labelBinaryPaths">
         <property name="text">
          <string>&lt;b&gt;Paths to program binaries&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QLabel" name="labelPreferredPaths">
         <property name="text">
          <string>&lt;b&gt;Preferred folders for input &amp;amp; output files&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="lineEditPreferredOutputPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Preferred folder to use for non-CHD output files</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEditPreferredCHDOutputPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Preferred folder to use for CHD output files</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEditPreferredInputPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Preferred folder to search for non-CHD input files</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEditPreferredCHDInputPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Preferred folder to search for CHD input files</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QToolButton" name="toolButtonBrowsePreferredCHDInputPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="icon">
          <iconset resource="qchdman.qrc">
           <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QToolButton" name="toolButtonBrowsePreferredInputPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="icon">
          <iconset resource="qchdman.qrc">
           <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QToolButton" name="toolButtonBrowsePreferredCHDOutputPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="icon">
          <iconset resource="qchdman.qrc">
           <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QToolButton" name="toolButtonBrowsePreferredOutputPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="icon">
          <iconset resource="qchdman.qrc">
           <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelChdmanWarningPixmap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="pixmap">
          <pixmap resource="qchdman.qrc">:/images/warning.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="labelPreferredCHDInputPath">
         <property name="text">
          <string>CHD input</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="labelPreferredInputPath">
         <property name="text">
          <string>Non-CHD input</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="labelPreferredCHDOutputPath">
         <property name="text">
          <string>CHD output</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="labelPreferredOutputPath">
         <property name="text">
          <string>Non-CHD output</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBoxLanguage</tabstop>
  <tabstop>comboBoxStyle</tabstop>
  <tabstop>fontComboBoxAppFont</tabstop>
  <tabstop>spinBoxAppFontSize</tabstop>
  <tabstop>fontComboBoxLogFont</tabstop>
  <tabstop>spinBoxLogFontSize</tabstop>
  <tabstop>fontComboBoxEditorFont</tabstop>
  <tabstop>spinBoxEditorFontSize</tabstop>
  <tabstop>checkBoxShowProjectHelp</tabstop>
  <tabstop>checkBoxMaximizeWindows</tabstop>
  <tabstop>checkBoxNativeFileDialogs</tabstop>
  <tabstop>checkBoxLogChannelNames</tabstop>
  <tabstop>lineEditChdmanBinary</tabstop>
  <tabstop>toolButtonBrowseChdmanBinary</tabstop>
  <tabstop>lineEditPreferredCHDInputPath</tabstop>
  <tabstop>toolButtonBrowsePreferredCHDInputPath</tabstop>
  <tabstop>lineEditPreferredInputPath</tabstop>
  <tabstop>toolButtonBrowsePreferredInputPath</tabstop>
  <tabstop>lineEditPreferredCHDOutputPath</tabstop>
  <tabstop>toolButtonBrowsePreferredCHDOutputPath</tabstop>
  <tabstop>lineEditPreferredOutputPath</tabstop>
  <tabstop>toolButtonBrowsePreferredOutputPath</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="qchdman.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
